### PR TITLE
Add Code Of Conduct

### PR DESCRIPTION
**Pull request by ivannieto**

### CODE REVIEW

**Comment by ivannieto** [comment]
Ok! 👍

**Comment by pauljherring** [comment]
It appears NodeBB isn't the only random project to have been targeted with  this nonsense by this user. A selection:

https://github.com/FreeCodeCamp/FreeCodeCamp/pull/12202
https://github.com/nodejitsu/node-http-proxy/pull/1119
https://github.com/rg3/youtube-dl/pull/11526
https://github.com/janl/mustache.js/pull/616
https://github.com/mrdoob/three.js/pull/10463
https://github.com/ratpack/ratpack/pull/1139

Basically he's made 29 pull requests today alone - to 29 different projects, of which 20 involved adding a code of conduct.

**Comment by ivannieto** [comment]
Addin the Code of Conduct for each project I've seen lacking it. Where is the problem of this? Where is the problem on doing 29 PR's in a day? Is there any limit that I'm not aware of? Is it a bad or invalid PR? Let me know.

**Comment by ivannieto** [comment]
@pauljherring, I understood really well the opinion by @julianlam on not using this CoC file, but not yours, at all. Please, let me know why I should act as you wish.

**Comment by pauljherring** [comment]
It's the apparent unwarranted assumption that you've made - that every project on github requires it.

On projects that don't, it comes across as trolling.

And the mass-adding it to unrelated projects that you otherwise don't appear to have contributed to, it comes across as bullying.

Both of which, unless I'm mistaken, appear in the so-called Code of Conduct you're shoving down people's necks.

**Comment by ivannieto** [comment]
And @pauljherring  is not a random project. It became as suggestion by 24pullrequests.com. Please review where is your project linked to, and act as you wish with the maintainers of this project, but not me.

**Comment by ivannieto** [comment]
It's a suggestion. It has been rejected. Nice. Is not a problem. But is a "nice to have". Nevermind. I don't care if you try to call me spammer. I still think is great, and I will try to push it wherever I think it's needed.

**Comment by ivannieto** [comment]
@pauljherring I really don't want to be rude, but you're the only one making assumptions in here.

**Comment by ivannieto** [comment]
And said that, I will leave this project.

I'm nevermore interested in it, due to this "conversation". I'm looking at interesting projects to start to contribute and learn from them, and adding a Code of Conduct makes sense for a lot of people. And for sure I've found great projects and great teams.

Judging someone by his/her acts without knowing a little shit about why he/she did it... pushes more people out than in.

Have a good Xmas.

**Comment by psychobunny** [comment]
It's ironic that a topic about code of conduct ended up so negatively 

I closed this PR because you don't have a GitHub account and can't sign our contributor license agreement. I also agree with Julian that this is probably obvious and doesn't need to be explicitly stated (in my opinion) 

Locking this thread, thanks for the original thought for this contribution
